src/main/python/linuxband/config.py

```diff
@@ -47,7 +50,7 @@ def load_config(self):
             try:
                 logging.debug("Opening config")
                 self.__config.read(Config.__rc_file)
-            except:
+            except IOError:
```

Comment:
OK, got it. But I still like to avoid empty except clauses. For any ConfigParser related call I would like to catch for `ConfigParser.Error`, the base class all exceptions thrown by ConfigParser.

Reply:
I'm not sure how you want to handle the ConfigParser.Error after you caught it. Could you provide a sample code? And what about other possible exceptions? In the future, if the ConfigParser introduces further exceptions, I would like to make sure we still catch them.

I agree that an empty except: clause is a bad code. How about just logging the exception:
except:
  logging.debug(e)